src/lib.rs

```diff
@@ -251,6 +249,9 @@ pub fn store<'a, T: Serialize>(
 ///
 /// [`store`]: fn.store.html
 pub fn store_path<T: Serialize>(path: impl AsRef<Path>, cfg: T) -> Result<(), ConfyError> {
+    let path = path.as_ref();
+    fs::create_dir_all(path.parent().unwrap()).map_err(ConfyError::DirectoryCreationFailed)?;
```

Comment:
Mind to add another commit that replaces the `unwrap()` call with an error? :smile:

Reply:
Cool, I replaced this panic with an error result.